scriptmodules/admin/stats.sh

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env bash
+
+# This file is part of The RetroPie Project
+#
+# The RetroPie Project is the legal property of its developers, whose names are
+# too numerous to list here. Please refer to the COPYRIGHT.md file distributed with this source.
+#
+# See the LICENSE.md file at the top-level directory of this distribution and
+# at https://raw.githubusercontent.com/RetroPie/RetroPie-Setup/master/LICENSE.md
+#
+
+rp_module_id="stats"
+rp_module_desc="Generates statistics about packages"
+rp_module_section=""
+
+function _dest_stats() {
+    local dest="$__tmpdir/stats"
+    mkUserDir "$dest"
+    echo "$dest"
+}
+
+function licences_stats() {
+    local idx
+    local mod_id
+    local mod_desc
+    local mod_section
+    local mod_licence
+
+    local dest="$(_dest_stats)/licences"
+    mkUserDir "$dest"
+
+    echo > "$dest/packages.csv"
+    for idx in ${__mod_idx[@]}; do
+        mod_section="${__mod_section[$idx]}"
+        mod_id="${__mod_id[$idx]}"
+        mod_desc="${__mod_desc[$idx]}"
+        mod_licence="${__mod_licence[$idx]}"
+        echo "$mod_section;$mod_id;$mod_desc;$mod_licence;" >> "$dest/packages.csv"
+    done
+
+    local git_hash=$(git -C "$scriptdir" log -1 --format=%h)
+    local git_date=$(git -C "$scriptdir" log -1 --format=%cd --date=iso-strict)
+    local git_branch=$(git -C "$scriptdir" rev-parse --abbrev-ref HEAD)
+    echo "$git_hash;$git_date;$git_branch;" > commit.csv
```

Comment:
This should be 
```
echo "$git_hash;$git_date;$git_branch;" > "$dest/commit.csv"
```

that's why the 'licences' page doesn't show the revision correctly.